B23 Ex03 Ronen 319047718 Ido 315942193/VehicleFactory.cs

```diff
@@ -210,15 +315,22 @@ private List<Wheel> createNewWheel(int i_NumOfWheels)
             return wheels;
         }
 
-        private Wheel createWheelFromUser()
+        private Wheel createWheelFromUser(float i_MaxPsi)
         {
-            Console.Write("Please insert the max PSI: ");
-            float userMaxPSI = float.Parse(Console.ReadLine());
             Console.Write("Please insert the manufaturer: ");
             string userManufaturer = Console.ReadLine();
             Console.Write("Please insert the current PSI: ");
```

Comment:
need to check also that user doesn't add negative PSI

Reply:
ADD